backend/context/config.py

```diff
@@ -61,3 +61,21 @@ def generate(index: int) -> typing.Tuple[LocalAccount, Config]:
         gas_currency_code=testnet.TEST_CURRENCY_CODE,
     )
     return (account, conf)
+
+
+def generate_premainnet(
+    index: int, vasp_private_key: str
+) -> typing.Tuple[LocalAccount, Config]:
+    port = 5000 + index
+    base_url = f"http://localhost:{port}/offchain"
+    account = LocalAccount.from_private_key_hex(vasp_private_key)
+    conf = Config(
+        wallet_custody_account_name=f"wallet{index}",
+        vasp_compliance_key=utils.private_key_bytes(Ed25519PrivateKey.generate()).hex(),
+        vasp_address=account.account_address.to_hex(),
+        base_url=base_url,
+        json_rpc_url="https://premainnet.diem.com/v1",
```

Comment:
shouldn't you read this from env variable?

Reply:
There is currently no convenient places what so ever to pull this value. It is the same problem as the premainnet chain ID, but even less likely to change. Seems good enough, for now. Any ideas?

Reply:
isn't this just the jsonrpc env variable?